config/initializers/govuk_notify.rb

```diff
@@ -16,8 +16,12 @@
     "55d35c86-7365-406b-909f-1b7b78529ea8",
   consent_school_subsequent_reminder_hpv:
     "6410145f-dac1-46ba-82f3-a49cad0f66a6",
-  session_clinic_initial_invitation: "fc99ac81-9eeb-4df8-9aa0-04f0eb48e37f",
-  session_clinic_subsequent_invitation: "eee59c1b-3af4-4ccd-8653-940887066390",
+  session_clinic_initial_invitation: "88d21cfc-39f6-44a2-98c3-9588e7214ae4",
```

Comment:
These template ID's are labelled as "[TEST]" on Gov UK Notify. I Think can stick with them and just remove "[TEST]" from the template name?